setup.py

```diff
@@ -39,7 +39,7 @@
 if sys.platform == 'darwin':
     # The clang compiler doesn't use --std=c++11 by default
     cpp_compile_args.append("--std=gnu++11")
-elif sys.platform == 'win32':
+elif sys.platform == 'win32' and "MSC" in sys.version:
```

Comment:
Thanks for this PR!

Using `platform.python_compiler()` would seem more robust, and is consistent with the use of platform above.

Reply:
Done.